### DISCUSSION TITLE
[FEATURE-REQUEST] En la sección reports agregar columnas subproperties

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#277] [3. Ideas & Feature Requests] [answered]
Hola

Me di cuenta en el momento de descargar una colección desde Reports me percaté que no vienen las subproperties asociadas a un  elemento de la colección. Creo que se debe considerar ya que en el futuro en el momento de actualizar una colección, el usuario debería saber las subproperties asociadas.

Gracias :>

Edit: Esto lo levanto solo para base de datos

**Comment by guillevalin** [comment, marked as answer]
Gracias por el feedback @NicolasPLG .

No está dentro de las prioridades a corto plazo, pero si esto es crítico y necesario para algún cliente, te solicitamos por favor contactarte con tu representante de Cotalker.
